documentation/en/default-lotus-config.toml

```diff
@@ -307,23 +292,62 @@
   # env var: LOTUS_EVENTS_MAXFILTERHEIGHTRANGE
   #MaxFilterHeightRange = 2880
 
-  # DatabasePath is the full path to a sqlite database that will be used to index actor events to
-  # support the historic filter APIs. If the database does not exist it will be created. The directory containing
-  # the database must already exist and be writeable. If a relative path is provided here, sqlite treats it as
-  # relative to the CWD (current working directory).
+
+[ChainIndexer]
+  # DisableIndexer controls whether the chain indexer is active.
+  # The chain indexer is responsible for indexing tipsets, messages, and events from the chain state.
+  # It is a crucial component for optimizing Lotus RPC response times.
+  # 
+  # Default: false (indexer is enabled)
+  # 
+  # Setting this to true will disable the indexer, which may significantly impact RPC performance.
+  # It is strongly recommended to keep this set to false unless you have a specific reason to disable it
+  # and fully understand the implications.
   #
-  # type: string
-  # env var: LOTUS_EVENTS_DATABASEPATH
-  #DatabasePath = ""
+  # type: bool
+  # env var: LOTUS_CHAININDEXER_DISABLEINDEXER
+  #DisableIndexer = false
```

Comment:
This should be set to True, as mentioned in: https://github.com/filecoin-project/lotus/pull/12421#issuecomment-2337466158

Reply:
@rjan90 Also wants to see what Rodd says here before we make a final call on this.

Reply:
But will this comment unresolved till we make a final decision here.

Reply:
> @rjan90 Also wants to see what Rodd says here before we make a final call on this.

https://filecoinproject.slack.com/archives/CP50PPW2X/p1725881194337459?thread_ts=1725868038.090649&cid=CP50PPW2X

Reply:
Similarly I think we should sync on the `EnableEthRPC` and `EnableActorEventsAPI` default configs. I´m not sure that defaulting these to `true` are ideal for most of Lotus' userbase.

Reply:
Agreed.  I don't think we need to rock the boat right now with turning things on by default.

Also, given the `ChainIndexer` context, maybe just call this "Enable"? (or maybe that isn't the code convention).  (i.e., `ChainIndexer.Enable = false`)

Reply:
Agreed given I am outvoted here. Reverted.

Reply:
If we're going with a default _OFF_ setting, then let's change it to `EnableIndexer` defaulting to `false`

Reply:
@rvagg This is done. The config field is now `EnableIndexer ` which defaults to false.